.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: kind/bug, lifecycle/needs-triage
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
Suggest we turn these into H2's that say (to the reader) what they are instead of using the imperative (to the writer).
```
## Short description
```

Reply:
👍